Description: backport of dpdk 16.04-rc1 fix for LP: #1559981

Forwarded: n/a (already upstream)
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2016-03-20

From da82ee17e6ea99bf2931383ac33b0caccaaaefce Mon Sep 17 00:00:00 2001
From: Jeff Shaw <jeffrey.b.shaw@intel.com>
Date: Mon, 8 Feb 2016 16:33:46 -0800
Subject: [PATCH] tools: fix unbinding failure handling

We should call sys.exit(), not divide sys by exit().

Signed-off-by: Jeff Shaw <jeffrey.b.shaw@intel.com>
Acked-by: Bruce Richardson <bruce.richardson@intel.com>
---
 tools/dpdk_nic_bind.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/dpdk_nic_bind.py b/tools/dpdk_nic_bind.py
index f02454e..a1923c5 100755
--- a/tools/dpdk_nic_bind.py
+++ b/tools/dpdk_nic_bind.py
@@ -317,7 +317,7 @@ def unbind_one(dev_id, force):
         f = open(filename, "a")
     except:
         print "Error: unbind failed for %s - Cannot open %s" % (dev_id, filename)
-        sys/exit(1)
+        sys.exit(1)
     f.write(dev_id)
     f.close()
 
-- 
2.7.3

